package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "screen-snippet",
+  "version": "0.0.0",
```

Comment:
can you set the version to 1.0.0?

Reply:
I'll do that once it is approved and we make a release